net-libs/libwebsockets/libwebsockets-2.1.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+http2 +ssl cgi access-log server-status client http-proxy ipv6 libev libressl libuv static-libs"
```

Comment:
Yes we have talked about this. It's my mistake that failed to include all these new etags.

`sqlite3`, `generic-sessions`, `lejp`, and `smtp` will be added.

`LWS_WITH_NO_LOGS`  should not be set otherwise compiling stage will fail.
Should `LWS_STATIC_PIC` ("Build the static version of the library with position-independent code") be an etag or left default?

`LWS_WITH_ESP8266` ("Build for ESP8266") is also skipped as it requires turning off a lot of options. Is that ok?